uaclient/cli/enable.py

```diff
@@ -449,6 +447,13 @@ def action_enable(args, *, cfg, **kwargs) -> int:
     enabled_services = _enabled_services(cfg).enabled_services
     all_dependencies = _dependencies(cfg).services
 
+    # On Focal, CIS transitioned to USG.
+    # If a user tries to enable CIS on Focal, they should get USG instead.
+    if "cis" in names and system.get_release_info().series == "focal":
+        entitlements_found.append("usg")
+        entitlements_not_found.remove("cis")
+        interactive_only_print(messages.CIS_IS_NOW_USG)
```

Comment:
I wonder if this logic should sit on the entitlements module instead. The rationale is that this should also apply to the enable API, which uses the `entitlement_factory` function. I know that this function is not used here, but I think the entitlement module should be the one aware of this situation, and not the CLI module

Reply:
The problem is the still somehow - as we need to do this during the enablement step, but not for other commands.
If we try to be always smart, the workaround will grow bigger than we want to maintain imho.
I'm leaning towards adding a similar snippet in the API call and live with it.

Reply:
I think that would be okay. I think ideally we would still install the cis packages on focal too, though, unless product has agreed that it's okay to break that?

Reply:
So I have changed the USG class to accomodate the message+packageset changes for CIS on Focal, and applied the logic for both enablement flows.